### PR TITLE
fix: update year in footer from 2022 to 2023

### LINKED ISSUE #97: Update website footer copyright year from 2022 to 2023
#### Current Behavior

Update website footer copyright year from 2022 to 2023

#### Screenshots/Logs
check3

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by kaushik-rishi**
**Notes for Reviewers**

This PR fixes #97 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Savio629** [comment]
LGTM @kaushik-rishi 👍

File: site/src/components/Footer/index.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ const Footer = () => {
         </div>
       </div>
       <div className="container flex copyright">
-        <div className="text">&copy; 2022-<script>document.write(new Date().getFullYear())</script> The Meshery Authors</div>
+        <div className="text">&copy; 2023-<script>document.write(new Date().getFullYear())</script> The Meshery Authors</div>
```
Comment (Chadha93) [inline site/src/components/Footer/index.js:211]: @kaushik-rishi This won't work, it's hardcoded, Please make it dynamic, hint - see the script next to it.

Comment (Chadha93) [reply site/src/components/Footer/index.js:211]: // @Savio629

Comment (Savio629) [reply site/src/components/Footer/index.js:211]: As the year was initially hardcoded, I didn't looked properly to the script.
@kaushik-rishi Please make the required changes.
If you face any issue try to ping me...

Comment (kaushik-rishi) [reply site/src/components/Footer/index.js]: @Chadha93 can you check the updated pr ?

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**